July Quarterly Update Blog (#51)
NCAR/GeoCAT: blog/_posts/2022-07-07-July-2022-update.md

quarterly update we made in early April:  

### Project Raijin

The collaboration between [Project Raijin](https://raijin.ucar.edu/) and 
DOE's SEATS project has released the first official version, 
[v2022.06.0](https://github.com/UXARRAY/uxarray/releases/tag/v.2022.6.0), of 
the [UXarray](https://uxarray.readthedocs.io/en/latest/index.html) package. 
Please see its [Conda distribution](https://anaconda.org/conda-forge/uxarray)
page as well.

Other than the release, there are other insights about the Project Raijin and 
UXarray that we'd like to share:

#### UXarray Discussions

Having input and feedback from our community is vital for the success of the 
UXarray package (so Project Raijin). We encourage our community members to share 
their feedback with us and give input to the UXarray project lifecycle. In 
accordance with this, we created a Github Discussion, [Prioritization of Uxarray 
analysis operators](https://github.com/UXARRAY/uxarray/discussions/46), to listen 
to our community's priorities with the data analysis functions to be implemented. 
The discussion has received a lot of feedback and led to conversation between the 

> folks. We are looking forward to implementing such functions as well as to keep 

Reviewer: ```suggestion
community members. We are looking forward to implementing such functions as well as to keep 

```
This edit could also encompass deleting "the" from the ending of line 36 so the sentence would read "...to conversation between [the] community members"

discussing with the community in the future.

#### How to contact us

If you would like to engage with Project Raijin and/or UXarray but are not
100% sure how, please see our
[Contact Us](https://raijin.ucar.edu/contact.html) page!

### GeoCAT Software Releases

We started to distribute the GeoCAT toolkits (GeoCAT-comp, GeoCAT-f2py, 
GeoCAT-datafiles, and GeoCAT-viz) on conda-forge channel in this quarter. 
Currently, we continue to distribute them via NCAR's Anaconda channel, too, but 
our plan is to use only the conda-forge channel.

Below you can find updates about our releases on several toolkits:

#### - GeoCAT-comp and GeoCAT-f2py April Releases (v2022.04.0)!
With these releases, there were some fixes such as making sure about the input 

> to output data type consistency, and some continuous integration improvements

Reviewer 2: ```suggestion
With these releases, there were some fixes such as ensuring input 
to output data type consistency, and some continuous integration improvements
```

This was a bit awkward to read before, but this small change should help

such as clarifying installation & updating instruction, and version update for 
some dependencies.

- Please see
  [GeoCAT-comp Releases](https://github.com/NCAR/geocat-comp/releases)
  for further information.

- Please see
  [GeoCAT-f2py Releases](https://github.com/NCAR/geocat-f2py/releases)
  for further information.

- In addition, please refer to
  [GeoCAT-comp API reference](https://geocat-comp.readthedocs.io/en/latest/api.html)
  for detailed information about the GeoCAT-comp functions and how GeoCAT-f2py
  is contained under GeoCAT-comp.

#### - GeoCAT-viz April and May Releases (v2022.04.0 and v2022.05.0)!
GeoCAT-viz has gone under a huge but backwards-compatible structural change 
in [v2022.04.0](https://github.com/NCAR/geocat-viz/releases/tag/v2022.04.0) 
and [v2022.05.0](https://github.com/NCAR/geocat-viz/releases/tag/v2022.05.0)

> that also has several new functionalities released! One of the biggest new 

Reviewer: ```suggestion
that also introduces several new functionalities! One of the major new 

```

functionalities is the capability to plot Taylor diagrams through a class 
with the same name. The structural changes include moving the Contour plot 
functionalities into a new class, Contour, which also enables tuning many 
Contour plot-related arguments at the time of initiation. This helps the user 

> avoid setting those arguments by calling several functions separately, so it 

Reviewer: ```suggestion
avoid setting those arguments by calling several functions separately, thus 

```

> helps reduce the lines of codes significantly.

Reviewer: ```suggestion
reducing the number of lines of code significantly.

```
